tox.ini

```diff
@@ -61,14 +61,49 @@ commands =
 ; Generates documentation with sphinx. There are other steps in the Github Actions workflow
 ; to publish the documentation on release.
 changedir = {toxinidir}
-deps = -rdocs/requirements-doc.txt
+deps = 
+    -rdocs/requirements-doc.txt
+    tensorflow 
+    torch 
+    torchmetrics
+    git+https://github.com/NVIDIA-Merlin/core.git
+    git+https://github.com/NVIDIA-Merlin/dataloader.git
+    git+https://github.com/NVIDIA-Merlin/models.git
+    git+https://github.com/NVIDIA-Merlin/NVTabular.git
+    git+https://github.com/NVIDIA-Merlin/systems.git
+    git+https://github.com/NVIDIA-Merlin/Transformers4Rec.git
+    
+allowlist_externals = 
+    git
+    cp
 commands =
+    git clone --depth 1 https://github.com/NVIDIA-Merlin/core.git ./docs/core
+    cp -r ./docs/core/docs/source/api ./docs/source/api_core/
+    git clone --depth 1 https://github.com/NVIDIA-Merlin/dataloader.git ./docs/dataloader
+    cp ./docs/dataloader/docs/source/api.rst ./docs/source/api_dataloader.rst
+    git clone --depth 1 https://github.com/NVIDIA-Merlin/models.git ./docs/models
+    cp ./docs/models/docs/source/api.rst ./docs/source/api_models.rst
+    git clone --depth 1 https://github.com/NVIDIA-Merlin/NVTabular.git ./docs/NVTabular
+    cp ./docs/NVTabular/docs/source/api.rst ./docs/source/api_nvtabular.rst
+    git clone --depth 1 https://github.com/NVIDIA-Merlin/systems.git ./docs/systems
+    cp ./docs/systems/docs/source/api.rst ./docs/source/api_systems.rst
+    git clone --depth 1 https://github.com/NVIDIA-Merlin/Transformers4Rec.git ./docs/Transformers4Rec
+    cp -r ./docs/Transformers4Rec/docs/source/api ./docs/source/api_transformers4rec/
     python -m sphinx.cmd.build -E -P -b html docs/source docs/build/html
```

Comment:
If `tox` alone can get you what you need, I think I'd just go with that.  (And, tbh, it escaped my imagination that `tox` would make it so simple.)

The only alternative that I'd consider is using one of the containers because the containers already have the tagged source in the container.

Reply:
I wasn't sure if the documentation is build inside a container - in addition, I want to pull the latest documentation code from every repository